cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/meta/RemoteNamespaceQueryHandler.java

```diff
@@ -35,24 +35,24 @@
 @Path(AbstractRemoteSystemOpsHandler.VERSION + "/execute")
 public class RemoteNamespaceQueryHandler extends AbstractRemoteSystemOpsHandler {
```

Comment:
The name is not good. Remote means it is talking to remote, but it seems like this one is actually not? Or a bit unclear.